architecture/core_objects/builds.adoc

```diff
@@ -171,55 +171,12 @@ You can define multiple Docker registry entries in this file. Alternatively, you
 can also add authentication entries to this file by running the `docker login`
 command. The file will be created if it does not exist.
 
-Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
-
-====
-----
-$ osc create -f secret.json
-----
-====
-
-The `*data*` field for the `*Secret*` object must contain the `*dockercfg*` key
-with the value set to the base64-encoded content of the *_.dockercfg_* file. For
-example:
-
-====
-
-----
-{
-  "apiVersion": "v1beta3",
-  "kind": "Secret",
-  "metadata": {
-    "name": "dockerhub"
-  },
-  "data": {
-    "dockercfg": "6yJodHRwc1ovL2zuZGV4LmRvY21lci5aby92MS8iOnsiYXV0aCI6ImJXWnZhblJwYXpwdVoybGxkR2d4TUE9PSIsImVtYWlsIj8ibWlAbWlmby5zayJ9fQ=="
-  }
-}
-
-----
-====
-
-Once you have the `*Secret*` created, you can add a `PushSecret` field into the
-`Output` section of the `BuildConfig` and set it to the name of the `*Secret*`
-that you created, which in the above example is `*dockerhub*`:
+To create the `*Secret*` resource from the `.dockercfg` file, you can run
+following command:
 
 ====
 ----
-"parameters": {
-  "output": {
-    "to": {
-      "name": "private-image"
-    },
-    "pushSecret":{
-      "name":"dockerhub"
-    }
-  }
-}
+$ openshift ex bundle-secret dockerhub ~/.dockercfg
```

Comment:
this should be moved up above a bunch of the stuff you nuked (and need to un-nuke)